salt/runners/manage.py

```diff
@@ -201,7 +201,7 @@ def up(tgt='*', tgt_type='glob', timeout=None, gather_job_timeout=None):  # pyli
     return ret
 
 
-def list_state(subset=None, show_ipv4=False, state=None):
+def list_state(subset=None, show_ip=False, state=None):
```

Comment:
This change will need to go through a deprecation process.

https://docs.saltstack.com/en/latest/topics/development/deprecations.html

Reply:
Where you have show_ipv4, and show_ip for 2 releases, and then it can be removed.